onnxruntime/contrib_ops/cuda/bert/relative_attn_bias_impl.cu

```diff
@@ -149,15 +148,73 @@ template Status LaunchRelPosAttnBiasKernel<half>(cudaStream_t stream,
                                                  const bool is_bidirectional,
                                                  const int max_threads_per_block);
 
-template <typename T>
+namespace {
+template <typename T, size_t size>
```

Comment:
This template looks similar to the one in decoder mha. Maybe we can refactor to a shared header file?

Reply:
sure, will send out another PR for refactoring